docs/001_develop/03_client-capabilities/008_interaction/dialog.mdx

```diff
@@ -0,0 +1,298 @@
+---
+title: 'Dialog'
+sidebar_label: 'Dialog'
+id: client-interaction-dialog
+keywords: [ interaction, interactive, active, window, overlay, popup]
+tags:
+- interaction
+- interactive
+- active
+- window
+- overlay
+- popup
+sidebar_position: 8
+---
+
+import DialogDemo from '/examples/ui/client-capabilities/interaction/dialog.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Anchors add different types of links to your page.
```

Comment:
Copy and paste error?

Reply:
This line is still there on the latest:
"Anchors add different types of links to your page"